From 8dc197721bd6e7bb68e57ddeff0abf1aac067a1f Mon Sep 17 00:00:00 2001
From: Michael Davis <mcarsondavis@gmail.com>
Date: Fri, 3 Nov 2023 19:34:54 -0500
Subject: [PATCH 027/188] Add an installation method field to the bug report
 template (#8711)

We can guess the installation method from the version tag and platform
in some cases but it would be useful to have this be explicit for the
sake of debugging packager-specific problems.
---
 .github/ISSUE_TEMPLATE/bug_report.yaml | 10 ++++++++++
 1 file changed, 10 insertions(+)

--- a/.github/ISSUE_TEMPLATE/bug_report.yaml
+++ b/.github/ISSUE_TEMPLATE/bug_report.yaml
@@ -56,6 +56,16 @@ body:
     validations:
       required: true
   - type: input
+    id: installation-method
+    attributes:
+      label: Installation Method
+      description: >
+        How you installed Helix - from a package manager like Homebrew or the
+        AUR, built from source, downloaded a binary from the releases page, etc.
+      placeholder: "source / brew / nixpkgs / flake / releases page"
+    validations:
+      required: true
+  - type: input
     id: helix-version
     attributes:
       label: Helix Version
